client/app/screens/LoginScreen.js

```diff
@@ -71,7 +72,10 @@ class LoginScreen extends React.Component {
 
       const response = await fetch(uri, init);
       const userProfile = await response.json();
-      userProfile = { ...userProfile, isAuthenticated: true }
+      const firebaseIdToken = await fetchIdToken(userProfile.firebaseCustomToken)
+        .catch(err => null);
```

Comment:
We'll need a better form of error handling here. There are a few instances where errors are "handled" but nothing really happens. I'm thinking of using something like [this](https://docs.nativebase.io/Components.html#Toast) to at least alert the user.

Reply:
Sounds good. It is at least logged in the implementation of `fetchIdToken`.